ouroboros-consensus-test/src/Test/Util/Orphans/DiffSeq/Arbitrary.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE StandaloneDeriving         #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Test.Util.Orphans.DiffSeq.Arbitrary () where
```

Comment:
Is it necessary to have this as a separate module, instead of having these instances in `Test.Util.Orphans.Arbitrary`?